tests/test_feature.rb

```diff
@@ -97,7 +85,13 @@ def test_nv_overlay
       assert_raises(Cisco::UnsupportedError) { Feature.nv_overlay_enable }
       return
     end
+    # Test if VxLAN can be configured
+    vxlan_linecard?
+    vdc_current = node.product_id[/N7/] ? vdc_lc_state : nil
+    vdc_lc_state('f3') if vdc_current
     feature('nv_overlay')
+    # Return testbed to pre-clean state
+    vdc_lc_state(vdc_current) if vdc_current
```

Comment:
You and Rob are both trying to fix this in different ways. See: https://github.com/cisco/cisco-network-node-utils/pull/383/files#diff-e295d4b860804d25a2520d0b081e221cR94

@robert-w-gries please coordinate with Aneesh on these changes.